src/rard/research/models/original_text.py

```diff
@@ -19,28 +20,38 @@ def related_lock_object(self):
     class Meta:
         ordering = ("citing_work", "reference_order")
 
+    @property
+    def reference_list(self):
+        """Returns a list of references with editor and ref position for each
+        unless there's only one, in which case only the ref position is shown"""
+        references = Reference.objects.filter(original_text=self)
+        if references.count() > 1:
+            return [
+                f"{reference.editor} {reference.reference_position} |"
+                for reference in references
+            ]
+        else:
+            return [references.first().reference_position]
```

Comment:
Please don't kill me, but this property could just return a string rather than a list right?
```suggestion
    def reference_list(self):
        """Returns a string which lists the editor and ref position for each reference
        unless there's only one, in which case only the ref position is shown"""
        references = Reference.objects.filter(original_text=self)
        if references.count() > 1:
            return " | ".join([
                f"{reference.editor} {reference.reference_position}"
                for reference in references
            ])
        else:
            return references.first().reference_position
```

Reply:
yes that would work